src/components/shared/Entity/EndpointConfig.tsx

```diff
@@ -42,19 +42,19 @@ function EndpointConfig({ connectorImage }: Props) {
         return <Error error={error} />;
     } else if (connector?.data) {
         return (
-            <>
-                <Typography variant="h5">Connection Config</Typography>
-                <Paper sx={{ width: '100%' }} variant="outlined">
+            <WrapperWithHeader
+                header={
```

Comment:
Made a reusable wrapper for the component that shows and header and wraps a body in a box.